contributing/README.md

```diff
@@ -104,20 +104,115 @@ Signed-off-by: Random J Developer <random@developer.example.org>
+that will help reviewees understand the context the reviewer is coming from.
+
+We encourage anyone in the community to conduct a code review on a PR.
+In most situations we prefer to have the following approvals before
+merging a PR:
+
+* At least one approval from [Reviewers]
+* At least one approval from [Maintainers]
+
+When opening a PR, GitHub will assign reviewers based on the project's
+code review settings and who gets assigned depends on what code the contributor
+changed, per [CODEOWNERS]. In most cases we expect that someone from
+[crossplane-reviewers] and a subject matter expert from
+[crossplane-maintainers] will be assigned.
+
+We encourage reviews from the community and [Reviewers] to take place before
+someone from the [Maintainers] group reviews a PR. This helps reduce the load
+on the project maintainers and ensures they can be more efficient in their
+reviews.
+
+### Expectation of PR Authors
+
+When your PR is ready for review please notify the assigned reviewers by
+by mentioning them in a comment.
+
+After implementing review feedback, the PR author should notify the reviewer
+by mentioning them in a comment when the PR is ready for another
+review.
+
+If you are not getting a response within a reasonable timeframe, remembering
+that reviewers and maintainers offer their time free of charge and have other
+obligations, you can reach out to the `#crossplane-owners` channel in the
+Crossplane community [Slack] workspace.
+
+### Expectation of Reviewers
+
+If you are assigned as a reviewer on a PR and you are unable to commit to
+reviewing the PR within a reasonable timeframe, you are encouraged to
+communicate this and manage the PR author's expectation.
+
+All reviewers are encouraged to consider the following aspects and to provide
+guidance to the PR author before giving their approval:
+
+* Is the code functionally correct?
+* Is the code written according to the [Coding Style](#coding-style)?
+* Is the solution idiomatically aligned with existing Crossplane APIs?
+* Is the code sufficiently covered by tests?
+* Has the PR author signed the DCO?
+* Are all CI jobs passing?
+
+When providing feedback please consider the following guidelines:
+
+* It helps the recipient a lot to be able to understand the context and
+intention behind comments.
+* Aim to provide feedback in a conversational style, rather than terse
+instructions.
+* Clearly articulate if you are sharing an opinion or instruction that
+needs to be complied with (i.e. a contribution guide rule).
+* Default to asking questions when things are not how we would expect them to
+be. Suggesting rather than demanding changes.
+* Proactively provide context when asking people to change things. Refer to
+where rules are defined or existing precedent exists, where possible.
+* Proactively clarify what needs to change for you to feel comfortable to
+approve a PR - which comments if any are blocking?
```

Comment:
Is this effectively the same as the third entry?

Reply:
You're right. I've combined the two into one.